BootcampKotlin/app/src/main/java/com/github/ybecker/bootcampKotlin/MainActivity.kt

```diff
@@ -0,0 +1,53 @@
+package com.github.ybecker.bootcampKotlin
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.activity.compose.setContent
+import androidx.appcompat.app.AppCompatActivity
+import androidx.compose.foundation.layout.Column
+import androidx.compose.material3.Button
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextField
+import androidx.compose.runtime.*
+import androidx.compose.ui.tooling.preview.Preview
+
+class MainActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
```

Comment:
You could have written that on a single line, so that your code looks more compact.